### DISCUSSION TITLE
pdf title too short

### DISCUSSION THREAD

**Discussion by WAAH1** [sumatrapdfreader/sumatrapdf#3197] [Q&A]
Hello,

when I have multiple pdf documents open, i always got difficulties to read the complete title of the document as the document name become shrunk. Even when i put the hand on the document tab, the complete document name doesn't appear.  

is there an easy way to have access to the **document name** when I have multiple pdf document open? 

![Capture](https://user-images.githubusercontent.com/110057376/209485047-cd50cfc7-4736-4f39-ab02-7fa6ca64ef85.JPG)

THANKS

**Comment by GitHubRulesOK** [comment]
current pre-release has tooltips
![image](https://user-images.githubusercontent.com/12754913/209487998-bf5d806b-6911-4acc-971d-b6010008419b.png)

**Reply by WAAH1** [reply]
thanks for your support

indeed it was because i was using and old version
